src/docfx/build/toc/TableOfContentsNode.cs

```diff
@@ -73,5 +73,12 @@ public TableOfContentsNode(TableOfContentsNode item)
             Children = item.Children;
             LandingPageType = item.LandingPageType;
         }
+
+        public TableOfContentsNode CloneWithoutItems()
```

Comment:
What is the difference between this method and the constructor in line 58?

Reply:
https://github.com/dotnet/docfx/pull/6579